huaweicloud/services/cce/resource_huaweicloud_cce_cluster_v3.go

```diff
@@ -151,6 +151,13 @@ func ResourceCCEClusterV3() *schema.Resource {
 				Computed:     true,
 				RequiredWith: []string{"eni_subnet_id"},
 			},
+			"enable_dist_mgt": {
```

Comment:
how about using `enable_remote_cluster` or `enable_distribute_management`?

Reply:
Done, it has been modified according to your suggestion.